devices/wholeBodyDynamics/KnownExternalWrench.h

```diff
@@ -0,0 +1,95 @@
+#ifndef KNOWNEXTERNALWRENCH_H
+#define KNOWNEXTERNALWRENCH_H
+
+#include <array>
+#include <vector>
+
+#include <iDynTree/Estimation/ExtWrenchesAndJointTorquesEstimator.h>
+//#include <iDynTree/Core/Position.h>
+//#include <iDynTree/Core/Direction.h>
+
+namespace wholeBodyDynamics
+{
+
+/**
+ * Class for processing the raw measure of
+ * a Six Axis Force Torque sensor.
+ *
+ * In particular this class process the raw measurement of a
+ * Six Axis FT sensor using an affine function:
+ *
+ * ftNew = secondaryCalibrationMatrix*ftOld - offset
+ *
+ * Where offset is the offset of the F/T sensor (tipically computed
+ * online at given instants where only one external force is known
+ * to be acting on the robot, while secondaryCalibrationMatrix is a
+ * secondary calibration matrix, that is tipically set to the identity.
+ *
+ */
```

Comment:
Carried over documentation?

Reply:
It was a TODO then I forgot about it, I'll update it